src/components/usersList/usersList.js

```diff
@@ -0,0 +1,26 @@
+import './usersList.scss';
+
+const CmpntName = ({ firstName = 'Philipp', lastName = 'Cherpakov', age = 26 }) => (
+  <div className="user">
+    <div className="f_name">{firstName}</div>
+    <div className="l_name">{lastName}</div>
+    <div className="age">{age}</div>
+  </div>
+);
+
+const ListCmpntName = ({ users = [] }) => (
```

Comment:
`CmpntName` do not give such name for component. If it is about users, call it `Users`. If it is a list, call it `List` etc.